vim25/soap/client.go

```diff
@@ -72,11 +72,29 @@ type Client struct {
 	Version   string // Vim version
 	UserAgent string
 
+	// ValidateRootCAs: if set to true, the soap client will ensure the root CA
+	// certificates are valid in SetRootCAs. If at least one of the certs is
+	// invalid, SetRootCAs will return an error.
+	// Defaults to false; which means that the client will silently ignore any
+	// cert validation errors during SetRootCAs.
+	ValidateRootCAs bool
```

Comment:
ok to leave the field for tests to use, let's make it unexported `validateRootCAs`.